Makefile

```diff
@@ -217,13 +217,13 @@ pao-functests-performance-workloadhints: cluster-label-worker-cnf pao-functests-
 pao-functests-performance-workloadhints-only:
 	@echo "Cluster Version"
 	hack/show-cluster-version.sh
-	hack/run-test.sh -t "test/e2e/performanceprofile/functests/0_config test/e2e/performanceprofile/functests/8_performance_workloadhints" -p "-v -r -timeout=5h --fail-fast --flake-attempts=2" -m "Running Functional WorkloadHints Tests" -r "--junit-report=report.xml"
+	hack/run-test.sh -t "test/e2e/performanceprofile/functests/0_config test/e2e/performanceprofile/functests/8_performance_workloadhints" -p "--v -r --fail-fast --flake-attempts=2 -timeout=5h --junit-report=report.xml" -m "Running Functional WorkloadHints Tests"
```

Comment:
what is the semantics of "-r"?

Reply:
Report parameters where positioned after the required test suites in the original scripts. 
When I wrote them with the rest of parameters it did not work for me, so I added a different parameter in the script to be able to write them after the suites as in the original.